### PR TITLE
RS23 rooms list support

### PR DESCRIPTION
**Pull request by Index01**
- changes for rs23 rooms list processing

### CODE REVIEW

File: backend/createStaffAndRooms.py [hunk 1]
```diff
@@ -44,74 +44,81 @@ def search_ticket(ticket, guest_entries):
...
-
-        if(elem["ROOMBAHT"]=="R"):
-            rooms.append(Room(name_hotel=elem['Room Type'].lstrip(),
-                             number=elem['Room'].lstrip(),
-                             available=True,
-                             name_take3=take3_name
-                             )
-                       )
+    rooms_rows = []
+    types2023RoomList = {
```
Comment (otakup0pe) [inline backend/createStaffAndRooms.py:50]: ten bux says i pull this out to a separate file once the dust has settled....

Comment (otakup0pe) [inline backend/createStaffAndRooms.py:50]: ten bux says i pull this out to a separate file once the dust has settled....

File: backend/createStaffAndRooms.py [hunk 2]
```diff
@@ -44,74 +44,81 @@ def search_ticket(ticket, guest_entries):
...
+            stripd = {k.lstrip().rstrip(): v.lstrip().rstrip() for k, v in row.items() if type(k)==str and type(v)==str}
+            rooms_rows.append(stripd)
+
+    for elem in rooms_rows:
+        if(elem["Placed By"]=="Roombaht"):
+            for roomtype in types2023RoomList:
+                if(elem["Room Type"] in roomtype[1]):
+                    setRoom = roomtype[0]
+                else:
+                    setRoom = "Room type not found"
```
Comment (otakup0pe) [inline backend/createStaffAndRooms.py:109]: Do we actually want to import stuff if it's a unknown room?

File: backend/createStaffAndRooms.py [hunk 3]
```diff
@@ -44,74 +44,81 @@ def search_ticket(ticket, guest_entries):
...
+                              name_hotel=elem['Room Type'],
+                              number=elem['Room'],
+                              available=True
+                              )
+                        )
         else:
             logger.debug(f'[-] Room excluded by ROOMBAHT colum: {elem}')
 
     logger.debug(f'swappable rooms: {rooms}')
+    print(f'rooms: {rooms}')
```
Comment (otakup0pe) [inline backend/createStaffAndRooms.py:121]: `logger.debug` plz

Comment (Index01) [reply backend/createStaffAndRooms.py:72]: deffff

File: backend/createStaffAndRooms.py (hunk 2 above)
Comment (Index01) [reply backend/createStaffAndRooms.py]: mmm prolly not :laughing:

File: backend/createStaffAndRooms.py (hunk 1 above)
Comment (Index01) [reply backend/createStaffAndRooms.py]: yesssss

**Pull request merged**